plugins/modules/rabbitmq_vhost.py

```diff
@@ -39,53 +39,171 @@
     type: str
     default: present
     choices: [present, absent]
-'''
+  login_user:
+      description:
+          - RabbitMQ user for connection.
+      type: str
+      default: guest
```

Comment:
```suggestion
      default: guest
      version_added: '1.5.0'
```
Please add it for all the new options